packages/contracts/src/RelayHub.sol

```diff
@@ -71,6 +72,11 @@ contract RelayHub is IRelayHub, Ownable {
         relayRegistrar = _relayRegistrar;
     }
 
+    function setBatchGateway(address _batchGateway) external onlyOwner {
+        require(batchGateway == address(0), "batchGateway already set");
```

Comment:
setting a batch gateway changes the security model of the RelayHub. it is not something that should be changeable, even with governance decision.
It should be settable in the constructor only, just like penalizer and stake manager.

Reply:
Fixed